app/services/webhooks/base_service.rb

```diff
@@ -16,6 +16,7 @@ def call
       payload = {
         :webhook_type => webhook_type,
         :object_type => object_type,
+        :organization_id => current_organization&.id,
```

Comment:
I don't think `current_organization` can be `nil` given we have `return if current_organization.webhook_endpoints.none?` above:

```suggestion
        :organization_id => current_organization.id,
```

Reply:
This is here because some models might now have the organization.

i've did the same as we have here
https://github.com/getlago/lago-api/blob/c64ff0186bc5bc08303163e15459c9de4ec7ae63/app/services/webhooks/base_service.rb#L53